concourse/model/traits/release.py

```diff
@@ -171,6 +171,14 @@ def _attribute_specs(cls):
         an optional prefix for release commit messages
         ''',
         type=str,
+    ),
+        AttributeSpec.optional(
+        name='next_cycle_commit_message_prefix',
```

Comment:
how about just calling this `bump_commit_message_prefix`?
I consider `next_cycle` to be less commonly understood

Reply:
as just discussed, for consistency reasons, let us opt for `nextversion_commitmsg_prefix`, or similar